composer.json

```diff
@@ -12,13 +12,13 @@
     ],
     "require": {
         "php": ">=5.4",
-        "symfony/framework-bundle": "~3.2",
-        "symfony/event-dispatcher": "~3.2",
+        "symfony/framework-bundle": "~2.8 || ~3.2",
+        "symfony/event-dispatcher": "~2.8 || ~3.2",
         "justinrainbow/json-schema": "~4.1"
     },
     "require-dev": {
-        "phpunit/phpunit": "~5.7.0",
-        "symfony/browser-kit": "~3.2"
+        "phpunit/phpunit": "~4.0.0 || ~5.7.0",
```

Comment:
i would use `~4.0` because `~4.0.0` will avoid `>4.1` to install same for `~5.7`